Assets/Scripts/P2PNetcodeSample/Networking/EOSTransportManager.cs

```diff
@@ -1279,6 +1289,7 @@ public bool TryReceivePacket(out ProductUserId remoteUserId, out string socketNa
         //
 
         private ulong ConnectionRequestNotificationsId = 0;
+        private ulong ConnectionEstablishedNotificationsId = 0;
```

Comment:
Field members should always be placed above any functions in a class - just as by convention.

Reply:
They should also have comments that sufficiently explain what they are.

I understand that in _this PR_ you just renamed the field member - but I think it is appropriate in this circumstance to take the opportunity to move the field member to the correct place and add documentation - even if that is every so slightly beyond the scope of the PR.